marin/run/vars.py

```diff
@@ -1,6 +1,19 @@
 # Environment variables that we want to set in Ray
 ENV_VARS = {
     "PYTHONPATH": "./submodules/levanter/src:${PYTHONPATH}",
+    "LIBTPU_INIT_ARGS": (
+        "--xla_tpu_scoped_vmem_limit_kib=81920 "
```

Comment:
i think this is too big for v4?

Reply:
copied directly from maxtext. Do we want to change it somehow?

Reply:
It seems like they didn't say that this is just for some specific kinds of tpus

https://github.com/AI-Hypercomputer/maxtext/blob/main/benchmarks/xla_flags_library.py#L33

Reply:
sure, different generations have different vmem limits, and this is too big for v4. v4 has only 16MiB vmem and this is 80

Reply:
hmmm ok so should we remove this line and let users put it if they care? idk how it will affect performance.

Reply:
I tried on v4 and it runs without any problem.